DisCatSharp/DiscordIntents.cs

```diff
@@ -133,7 +140,7 @@ public enum DiscordIntents
 		GuildPresences = 1 << 8,
 
 		/// <summary>
-		/// Whether to include guild message events.
+		/// Whether to include guild message events. Note that you may receive empty contents if you don't have the message content intent.
```

Comment:
Same thing here and above, should probably add a note saying this will only affect verified bots. Might otherwise make people more apprehensive towards upgrading to v10.